### PR TITLE
Implementation Sszs - Muka Signal

### PR DESCRIPTION
**Pull request by TruongQuangSB**

### CODE REVIEW

File: java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sszs/SszsTransformator.xtend [hunk 1]
```diff
@@ -0,0 +1,736 @@
...
+				),
+				new Case<ETCS_Signal>(
+					[
+
+						signalRealAktivShirmArt ===
+							ENUM_SIGNAL_ART_SPERRSIGNAL && !
+								IDSignal?.value.
+							isStartOrDestinationOfAnyTrainRoute
+					],
+					["sLs"]
```
Comment (pstuecker) [inline java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sszs/SszsTransformator.xtend:159]:
```suggestion
					["Ls"]
```

File: java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sszs/SszsTransformator.xtend [hunk 2]
```diff
@@ -0,0 +1,736 @@
...
+				[ETCSSignalTBV?.TBVMeldepunkt?.wert.translate]
+			)
+
+			// AB: Sszs.TBV.Laenge_Tunnelbereich
+			fill(
+				row,
+				cols.getColumn(Laenge_Tunnelbereich),
+				etcsSignal,
+				[
+					(ETCSSignalTBV?.TBVTunnelbereichLaenge?.wert ?: "").toString
```
Comment (pstuecker) [inline java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sszs/SszsTransformator.xtend:543]: This should be rounded up to an integer

File: java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sszs/SszsTransformator.xtend [hunk 3]
```diff
@@ -0,0 +1,736 @@
...
+				],
+				MIXED_STRING_COMPARATOR
+			)
+
+			// AF: Sszs.Bemerkung
+			fillSwitch(
+				row,
+				cols.getColumn(Bemerkung),
+				refSignal,
+				new Case<Signal>(
```
Comment (pstuecker) [inline java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sszs/SszsTransformator.xtend:591]: If both cases apply, both should be returned

**Review by pstuecker — CHANGES_REQUESTED** [review]

File: java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sszs/SszsTransformator.xtend (hunk 1 above)
Comment (TruongQuangSB) [reply java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sszs/SszsTransformator.xtend:160]: Why? In this case should fill 'sLs'

**Review by sbx320 — APPROVED** [review]

**Review by pstuecker — APPROVED** [review]

**Pull request merged**